src/libstd/process.rs

```diff
@@ -552,6 +552,15 @@ impl Command {
 
     /// Configuration for the child process's standard input (stdin) handle.
     ///
+    /// Defaults to [`inherit`] when used with `spawn` or `status`, and
+    /// defaults to [`piped`] when used with `output`.
+    ///
+    /// On Windows, if the `#![windows_subsystem = "windows"]` attribute is
+    /// set, no stdin is connected unless explicitly assigned.
```

Comment:
out of curiosity, what do you mean by 'assigned' here? i'm guessing that's some windows-specific concept i'm not familiar with